extension/src/extension.ts

```diff
@@ -0,0 +1,19 @@
+import * as vscode from 'vscode';
+import { AtlasClassCompletionItemProvider } from './completion-item-provider';
+
+export async function activate(context: vscode.ExtensionContext) {
+	const documentSelectors = [
+		{ language: 'typescript', scheme: 'file' },
```

Comment:
Just want to verify that this is how it is intended to work. 

If I start typing a class name such as `padding` in a TS file, I get the Intellisense. 
![image](https://user-images.githubusercontent.com/94572161/193934939-62388df2-fe18-4dfa-80fe-e4a4ca3688ec.png)

If I try to do the same inside quotations, the intellisense does not appear. 
![image](https://user-images.githubusercontent.com/94572161/193935025-dc2fa2b8-16c9-4aa1-a6e5-7371738eb992.png)

Reply:
Definitely peculiar it may have to do with the cursor context. I've tested and it does not always work, but it does depend on the current word. If you type with the cursor directly next to the quote, it doesn't work. 

<img width="602" alt="image" src="https://user-images.githubusercontent.com/30843002/193939229-a78851c3-a180-4def-9b71-a68176ec1267.png">

Reply:
We stayed clear of doing anything too intense regarding cursor context, but that is an option for future work